docs/dev.md

```diff
@@ -0,0 +1,15 @@
+#### dev
+
+----
+Start a Liberty server in dev mode.
+
+
+Dev mode provides three key features. Code changes are immediately detected, recompiled, and picked up by your running server. Unit and integration tests are run on every code change to give you instant feedback on the status of your code. Finally, it allows you to attach a debugger to the running server at any time to step through your code.
+
+
+| Parameter | Description | Default Value |
```

Comment:
To be consistent with the other docs, use a Required column instead of a Default Value column.  You can describe the default value in the description if appropriate.